README.md

```diff
@@ -45,91 +33,77 @@ As a Maker
 
-Notes on functionality:
-------
+### Domain diagram
+![user story](/user_story.png)
 
-* You don't have to be logged in to see the peeps.
-* Makers sign up to chitter with their email, password, name and a username (e.g. samm@makersacademy.com, password123, Sam Morgan, sjmog).
-* The username and email are unique.
-* Peeps (posts to chitter) have the name of the maker and their user handle.
-* Your README should indicate the technologies used, and give instructions on how to install and run the tests.
+## How to use
 
-Bonus:
------
+### Git clone
 
-If you have time you can implement the following:
+git clone git@github.com:bear99a9/chitter-challenge.git
 
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
+cd chitter-challenge
 
-And/Or:
+### To set up the database
 
-* Work on the CSS to make it look good.
+Connect to `psql` and create the `chitter` database & the `chitter_test:` database;
 
-Good luck and let the chitter begin!
+```
+CREATE DATABASE chitter;
+CREATE DATABASE chitter_test;
+```
 
-Code Review
------------
+To set up the appropriate tables, connect to each database in `psql` and run the SQL scripts in the `db/migrations` folder in the given order.
 
-In code review we'll be hoping to see:
+### To run the Chitter app:
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+```
+bundle
+```
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want at this moment.
+```
+rackup -p 3000
+```
 
-Automated Tests:
------
+To view chitter, navigate to `localhost:3000/hompage`.
 
-Opening a pull request against this repository will will trigger Travis CI to perform a build of your application and run your full suite of RSpec tests. If any of your tests rely on a connection with your database - and they should - this is likely to cause a problem. The build of your application created by has no connection to the local database you will have created on your machine, so when your tests try to interact with it they'll be unable to do so and will fail.
+### To run tests:
 
-If you want a green tick against your pull request you'll need to configure Travis' build process by adding the necessary steps for creating your database to the `.travis.yml` file.
+```
+rspec
+```
 
-- [Travis Basics](https://docs.travis-ci.com/user/tutorial/)
-- [Travis - Setting up Databases](https://docs.travis-ci.com/user/database-setup/)
+### To run linting:
 
-Notes on test coverage
-----------------------
+```
+rubocop
+```
 
-Please ensure you have the following **AT THE TOP** of your spec_helper.rb in order to have test coverage stats generated
-on your pull request:
+### Current Issues
```

Comment:
nice to keep track of what you need to work on - allows anyone looking at your github to suggest tips maybe?